http-ballerina-tests/build.gradle

```diff
@@ -26,6 +29,7 @@ def tomlVersion = project.version.replace("${snapshotVersion}", "")
 def ballerinaConfigFile = new File("$project.projectDir/Ballerina.toml")
 def ballerinaDependencyFile = new File("$project.projectDir/Dependencies.toml")
 def artifactCodeCoverageReport = file("$project.projectDir/target/cache/tests_cache/coverage/ballerina.exec")
+def artifactJar = file("$project.projectDir/build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${project.ballerinaLangVersion}/repo/cache/ballerina/http/${tomlVersion}/java11/")
```

Comment:
Is this the correct path?

Reply:
We can try copying the `cache` directory also here and assing the original location of the jar to `artifactJar` https://github.com/ballerina-platform/module-ballerina-http/blob/master/http-ballerina-tests/build.gradle#L97

Reply:
I tried the approach but it didn't work. gave the same result as using the `http.jar` in `http-ballerina` without copying cache.